api/src/test/java/org/openmrs/test/jupiter/BaseTestConfig.java

```diff
@@ -0,0 +1,92 @@
+package org.openmrs.test.jupiter;
+
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.MountableFile;
+
+import liquibase.integration.spring.SpringLiquibase;
+
+@Configuration
+@EnableTransactionManagement
+public class BaseTestConfig {
+	
+	@Autowired
+	private DataSource dataSource;
+	
+	private static final String PASSWORD = "admin";
+	
+	private static final String USER = "postgres";
+	
+	private static final String DB_NAME = "openmrs";
+	
+	private static LocalContainerEntityManagerFactoryBean emf;
+	
+	private static JpaTransactionManager transactionManager;
+	
+	private static PostgreSQLContainer postgreSqlContainer = new PostgreSQLContainer("postgres:13");
+	
+	public SpringLiquibase testLiquibase() {
+		SpringLiquibase liquibase = new SpringLiquibase();
+		liquibase.setChangeLog("classpath:liquibase/liquibase-test-insert-with-uuid.xml");
+		liquibase.setDataSource(dataSource);
+		return liquibase;
+	}
+	
+	public DataSource testDataSource() {
```

Comment:
are you trying to test a data source or create a data source?

Reply:
> are you trying to test a data source or create a data source?

Ohh sorry i should be creating data source. Will rectify that. Thanks.